### PR TITLE
Use specific version of the bcinsider artifact

### PR DESCRIPTION
**Pull request by mazhelez**
- Use specific version of the _bcinsider_ artifact and add a workflow to regularly uptake the newest version
- Use app baselines from a BC artifact
- Disable breaking checks for Clean mode as it's failing (known issue)
- Disable Translated mode for System Application as it's failing due to an issue with translations

### CODE REVIEW

File: Build/Scripts/EnlistmentHelperFunctions.psm1 [hunk 1]
```diff
@@ -116,24 +116,60 @@ function Set-ConfigValue() {
...
 .Parameter PackageName
     The name of the package
-.Parameter MaxVersion
-    The maximum version to look for
 #>
 function Get-PackageLatestVersion() {
     param(
         [Parameter(Mandatory=$true)]
-        [string] $PackageName,
-        [string] $MaxVersion
```
Comment (mazhelez) [inline Build/Scripts/EnlistmentHelperFunctions.psm1:126]: Max version should probably stay.

File: Build/Scripts/EnlistmentHelperFunctions.psm1 [hunk 2]
```diff
@@ -116,24 +116,76 @@ function Set-ConfigValue() {
...
+function Get-LatestBCArtifactVersion
+(
+    [Parameter(Mandatory=$true)]
+    $minimumVersion
+)
+{
+    $artifactUrl = Get-BCArtifactUrl -type Sandbox -country base -version $minimumVersion -select Latest
+
+    if(-not $artifactUrl) {
+        #Fallback to bcinsider. Should go away soon. Haha, yeah right.
```
Comment (mazhelez) [inline Build/Scripts/EnlistmentHelperFunctions.psm1]: Fix!

File: Build/Scripts/EnlistmentHelperFunctions.psm1 (hunk 1 above)
Comment (mazhelez) [reply Build/Scripts/EnlistmentHelperFunctions.psm1:126]: Not really, no.

File: .github/workflows/UpdateBCArtifactVersion.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      UpdateBranches: ${{ steps.GetBranches.outputs.UpdateBranches }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Get Branches
+        id: GetBranches
+        run: |
+          git fetch
+          $branches = @(git for-each-ref --format="'%(refname:short)'" refs/remotes/origin/release/ | % { $_ -replace 'origin/', '' })
```
Comment (aholstrup1) [inline .github/workflows/UpdateBCArtifactVersion.yaml:26]: I think for the sake of keeping it consistent, I think we should name our branches releases/xxx

Comment (mazhelez) [reply .github/workflows/UpdateBCArtifactVersion.yaml:26]: I thought _release**s**/22_ doesn't work with AL-Go?

Comment (mazhelez) [reply .github/workflows/UpdateBCArtifactVersion.yaml:26]: Anyway, let's have at it separately (not this PR).

**Pull request merged**